### ISSUE TITLE
Issue with amber in the Mine of Dor Kiredon

### ISSUE THREAD

**Issue by macce2** [Pyrdacor/Ambermoon#103]
You get the message "There's a piece of amber";
but nothing does show up on the treasure screen
which appears.
I was able to repeat the case.
Tested with Ambermoon Advanced v. 1.03 English on real Amiga.

**Comment by macce2** [comment]
I must add the potentially relevant fact that when entering this area in the game;
one character of the paty did already have about 60 (!) pieces of Amber;
it's a pity that I can't recall the exact place that they were found (!);
but it was on Kire's moon..
60 pieces of amber is quite much I think.. (!?!)

**Comment by macce2** [comment]
Oh excuse me, the amount of amber pieces I found in the game before
the issue, does seem to be valid (!?!):
even though IMHO, the total amount of the amber pieces available 
maybe could be slightly reduced.

**Comment by Pyrdacor** [comment]
Yes the amount is quite high. I guess it is because you need 1 piece to craft a nav stone and you can actually craft a lot of those.

I won't change it in the original but maybe in the advanced version. Maybe it's better to sell them at some later point instead of placing them everywhere.

**Comment by Pyrdacor** [comment]
> You get the message "There's a piece of amber"; but nothing does show up on the treasure screen which appears. I was able to repeat the case. Tested with Ambermoon Advanced v. 1.03 English on real Amiga.

Can you tell me where this message appeared exactly?

**Comment by macce2** [comment]
In the mine of Dor Kiredon; everywhere that there is (should be) amber pieces to be found.
At the first floor, the place that theres " a garbage" sign marked at the map view; was the place that I found the issue first.

**Comment by macce2** [comment]
![20230916_041203](https://github.com/Pyrdacor/Ambermoon/assets/116389369/18fcf3c5-8764-4c51-90b5-58f5ff891777)
